test/integration/examples/apiserver_test.go

```diff
@@ -90,9 +92,14 @@ func TestAggregatedAPIServer(t *testing.T) {
 			"--kubeconfig", wardleToKASKubeConfigFile,
 		})
 		if err := wardleCmd.Execute(); err != nil {
-			t.Fatal(err)
+			errs <- err
 		}
 	}()
+	// wait for it
+	err = <-errs
```

Comment:
blocking here until the server stops defeats the purpose of starting it in a goroutine

Reply:
Will remove the channel and change it to regular error message.